aptos-move/aptos-release-builder/data/release.yaml

```diff
@@ -23,7 +23,21 @@ proposals:
     execution_mode: MultiStep
     update_sequence:
     - Execution:
-        V2:
+        V3:
           transaction_shuffler_type:
             deprecated_sender_aware_v1: 32
           block_gas_limit : 35000
+          transaction_deduper_type: txn_hash_and_authenticator_v1
+  - name: enable_transaction_shuffling
+    metadata:
+      title: "Enable Transaction Shuffling"
+      description : "AIP-27: Sender Aware Transaction Shuffling"
+      discussion_url: "https://github.com/aptos-foundation/AIPs/issues/109"
+    execution_mode: MultiStep
```

Comment:
why is it a multi- step when there's only one step?

Reply:
I don't understand what this is, just copied the gas limit one 😂

Reply:
Changed this to `SingleStep`